src/dev.rs

```diff
@@ -139,4 +139,12 @@ impl crate::body::MessageBody for AnyBody {
             AnyBody::Boxed { body } => body.as_pin_mut().poll_next(cx),
         }
     }
+
+    fn try_into_bytes(self) -> Result<crate::web::Bytes, Self> {
+        match self {
+            AnyBody::None => Ok(crate::web::Bytes::new()),
```

Comment:
I don't think this s right . My argument is `AnyBody::None.size() != Bytes::new().size()`. This also needs to be consistent with `body::None`.

Reply:
yeah, whatever the behavior the None variants should align with `body::None` I suggest handling this by wrapping that struct so that any future tweaks affect all places